api_docs/unmerged.d/ZF-8d0c68

```diff
@@ -0,0 +1,6 @@
+* [`GET /invites`](/api/get-invites): The `stream_ids` and
+  `include_realm_default_subscriptions` field has been added
+  to be returned for all invites.
+* [`PATCH /invites/multiuse/{invite_id}`](/api/edit-invite-link):
```

Comment:
This part of allowing editing invites should be added in the next commit in a separate file using the same process that you used here.